### PR TITLE
[Asset] removed unused private property.

### PR DESCRIPTION
**Pull request by hhamon**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by WouterJ** [comment]
Looks correct to me.

Status: Reviewed

File: src/Symfony/Component/Asset/UrlPackage.php [hunk 1]
```diff
@@ -62,7 +61,7 @@ public function __construct($baseUrls = array(), VersionStrategyInterface $versi
         $sslUrls = $this->getSslUrls($baseUrls);
 
         if ($sslUrls && $baseUrls !== $sslUrls) {
-            $this->sslPackage = new UrlPackage($sslUrls, $versionStrategy);
+            $this->sslPackage = new self($sslUrls, $versionStrategy);
```
Comment (xabbuh) [inline src/Symfony/Component/Asset/UrlPackage.php]: Is that necessary?

Comment (WouterJ) [inline src/Symfony/Component/Asset/UrlPackage.php]: No, but it's easier when renaming the class name and apart from that, works exactly the same.

Comment (hhamon) [inline src/Symfony/Component/Asset/UrlPackage.php]: Fabbot.io forced to me to change me to `new self` to make the build pass. Looks like that's one of our coding conventions :)

**Comment by dunglas** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @hhamon.

**Pull request merged**